pglookout/cluster_monitor.py

```diff
@@ -155,6 +155,29 @@ def connect_to_cluster_nodes_and_cleanup_old_nodes(self):
         for instance, connect_string in self.config.get("remote_conns", {}).items():
             self._connect_to_db(instance, dsn=connect_string)
 
+    def _fetch_replication_slot_info(self, instance, cursor):
```

Comment:
A list of objects with many keys and values of various types is worth a type hint and a dataclass for the return type :)

Reply:
May look a bit weird, as it is the only type hint and dataclass usage in pglookout then, but sure. :-)